Sources/MarathonCore/ScriptManager.swift

```diff
@@ -218,6 +214,21 @@ public final class ScriptManager {
         }
     }
 
+    @discardableResult
+    private func saveFile(from url: URL, destinationDirectory: Folder, fileName: String) throws -> File {
```

Comment:
How about using the external parameter label `to` instead of `destinationDirectory`? That way the call site will read really nicely:
```
saveFile(from: url, to: folder, fileName: name)
```

Reply:
Oh dear, that is a lot cleaner. On it!